build.py

```diff
@@ -1740,11 +1751,18 @@ def __check_vs(self, opts):
         if not os.path.exists(vcvars_bat):
             raise Exception("'%s' could not be found. Please check you have Visual Studio installed at '%s' and that it supports the target platform '%s'." % (vcvars_bat, opts.vs_install_path, opts.platform))
 
+        # Add to the environment the gtk paths so meson can find everything
+        self.add_env('INCLUDE', self.gtk_dir + r'\include')

```

Comment:
sorry to not point out this before, but can we use os.path.join instead of that +?